src/frontends/paddle/src/frontend.cpp

```diff
@@ -135,22 +135,32 @@ bool normalize_framework_node(const std::shared_ptr<FrameworkNode>& node,
     return true;
 }
 
-std::istream* variant_to_stream_ptr(const ov::Any& variant, std::ifstream& ext_stream) {
+OPENVINO_SUPPRESS_DEPRECATED_START
+void variant_to_stream_ptr(const ov::Any& variant, std::stringstream& ss) {
     if (variant.is<std::istream*>()) {
-        return variant.as<std::istream*>();
+        ss << variant.as<std::istream*>()->rdbuf();
+    } else if (variant.is<std::shared_ptr<ngraph::runtime::AlignedBuffer>>()) {
+        auto& aligned_weights_buffer = variant.as<std::shared_ptr<ngraph::runtime::AlignedBuffer>>();
+        ss.write(aligned_weights_buffer->get_ptr<char>(), aligned_weights_buffer->size());
```

Comment:
It will bring extra memory copying, which may impact performance.
Could we avoid memory copying?

Reply:
@riverlijunjie it is a trade-off here.
Clean code style may conflict with performance in this case because this function deals with several different IO data type.
So, it is hard to meet the two requirements at the same time.

Reply:
@riverlijunjie any idea to share the AlignedBuffer's memory to avoid copy?